code/modules/clothing/suits/armor.dm

```diff
@@ -182,6 +182,38 @@
 		return 1
 	return 0
 
+/obj/item/clothing/suit/armor/reactive/teleport
```

Comment:
This is great and all, but you should remove the code you copypasta'd from the parent type.